### PR TITLE
Update README for conventional-changelog-conventionalcommits with example

### PR DESCRIPTION
**Pull request by yairkukielka**
It was hard for me to understand how all pieces of conventional changelog play together. It would be very useful to provide a specific example for users that want to customize this great preset

### CODE REVIEW

File: packages/conventional-changelog-conventionalcommits/README.md [hunk 1]
```diff
@@ -6,6 +6,40 @@ A concrete implementation of the specification described at
 [conventionalcommits.org](https://conventionalcommits.org/) for automated
 CHANGELOG generation and version management.
 
+
+## Indirect Usage (as preset)
+
+Use the [Concentional Changelog CLI Quick Start](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-cli#quick-start) with the `-p conventionalcommits` option.
```
Comment (bcoe) [inline packages/conventional-changelog-conventionalcommits/README.md:12]: typo, `Concentional` to `Conventional`.

Comment (yairkukielka) [reply packages/conventional-changelog-conventionalcommits/README.md]: fixed

**Comment by yairkukielka** [comment]
@bcoe could you take another look? Thank you!

**Comment by bcoe** [comment]
@yairkukielka I'm landing this, as it's a significant improvement to what we have, it might be worth however pointing people towards [standard-version](https://github.com/conventional-changelog/standard-version) and [release-please](https://github.com/googleapis/release-please) somewhere in the doc.

These are both concrete implementations of the conventionalcommits preset.

**Pull request merged**